projects/wave-core-new/src/lib/datatable/table/full-display/full-display.component.scss

```
@@ -0,0 +1,12 @@
+table {
+    margin: 1em;
+    border-collapse: collapse;
+    border-style: hidden;
+}
+
+table td {
+    border: 1px solid black;
+    border-top: 0px #000;
+    border-bottom: 0px #000;
+    padding: 5px;
+}
```

```suggestion
    padding: .5rem;
}

```

Try to use rather `rem` than px.